webpack.config.js

```diff
@@ -75,6 +75,7 @@ module.exports = {
 		'./assets/js/amp-validation-single-error-url-details-compiled': './assets/src/amp-validation-single-error-url-details.js',
 		'./assets/js/amp-story-editor-blocks-compiled': './assets/src/amp-story-editor-blocks.js',
 		'./assets/js/amp-story-blocks-compiled': './assets/src/blocks/stories.js',
+		'./assets/js/amp-agnostic-blocks-compiled': './assets/src/blocks/amp-agnostic.js',
```

Comment:
Maybe creating a new built JS file isn't needed for this. But it sounds like this block needs to be available in non-AMP, while `amp-blocks-compiled.js` is only available in Native AMP.